optaweb-vehicle-routing-frontend/src/store/route/operations.ts

```diff
@@ -32,6 +39,18 @@ export const deleteLocation: ThunkCommandFactory<number, DeleteLocationAction> =
     client.deleteLocation(locationId);
   });
 
+export const addVehicle: ThunkCommandFactory<void, AddVehicleAction> = (
+  () => (dispatch, getState, client) => {
+    dispatch(actions.addVehicle());
```

Comment:
The `AddVehicleAction` means that the process of adding a vehicle has started because the user has clicked the Add button. This can be later used to create a placeholder for the new vehicle until the operation is finished on backend and the new vehicle is received. Currently the action has no effect.

I agree that 1) the name is misleading and 2) it's an unused code. I'll keep it this way because addressing this is out of scope of this PR and to keep it consistent with other operations. Filed #146.